src/Projects/projects.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import './projects.css';
+import ProjectCard from './project-card';
+
+const data = [
+    {
+        title: 'Money Jars',
+        live: 'https://money-jars.herokuapp.com/',
+        github: 'https://github.com/Carenelizabeth/money-jars-client',
+        text: 'I needed a way for my kids to manage their money that was as awesome as the budgeting software that I personally use. I utilized its public API to allow me to add funds on my end while they can get granular with their savings goals on theirs.',
+    },
+    {
+        title: 'Something New',
+        live: 'http://somethingnew.carenkeyes.com/',
+        github: 'https://github.com/Carenelizabeth/something-new',
+        text: 'Have you ever been looking for something new and have no idea where to start? Wouldn’t it be great to read the highest-rated reviews for an area, rather than having to search for a business first? Something New combines data from CityGrid and Google maps to let you do just that.',
+    },
+    {
+        title: 'Family Reunion',
+        live: 'https://family-reunion-2018.herokuapp.com/',
+        github: 'https://github.com/Carenelizabeth/family-reunion',
+        text: 'Sometimes it feels like we spend most of our family get-togethers in a hotel room trying to decide what to do next. Family Reunion was born from a desire for a large group of people to easily decide who wants to do what. One participant posts an activity and others can join in or discuss.',
+    }
+]
+
+export default function Projects(props){
+    return(
+        <div className='container projects-wrapper'>
+        <ProjectCard
```

Comment:
I'd suggest moving  the `image` `alt` `stackURL` and `className` props into the JSON data you have above. Then you can render these like

```js
{data.map(function (card) {
    return <ProjectCard key={card.title} {...card} />;
})}
```
This says for each item in your `data` array, transform it to a ProjectCard (`map` takes an array and makes a new array by transforming each item). React prefers to have keys to render an array of components, so we use the `title` as the `key`.